cmd/config-v19.go

```diff
@@ -61,12 +61,19 @@ func (s *serverConfigV19) GetVersion() string {
 	return s.Version
 }
 
-// SetRegion set new region.
-func (s *serverConfigV19) SetRegion(region string) {
+// SetRegion set new region. SetRegion retruns previous region.
+func (s *serverConfigV19) SetRegion(region string) (prevRegion string) {
```

Comment:
Since `prevRegion` returned is not used at any of the call-sites, it appears unnecessary to add this change in.